src/deploy/osp_deployer/director.py

```diff
@@ -1408,6 +1427,7 @@ def delete_overcloud(self):
         for node in ls_nodes:
             node_state = node.split("|")[5]
             node_id = node.split("|")[1]
+            nod_id = node.split("|")[1]
```

Comment:
Why is this needed?  If needed, I recommend changing the name as the name is too similar to node_id.